packages/scripts/src/upgrade-tests/voting2/1_Propose.ts

```diff
@@ -204,23 +215,37 @@ async function main() {
     const iface = new hre.ethers.utils.Interface(getAbi("MultiRole"));
     const data = iface.encodeFunctionData("resetMember", [0, votingUpgrader.address]);
     adminProposalTransactions.push({ to: contractAddress, value: 0, data });
-    console.log(`2.i.  Multirole: transfer owner role of ${contractName} to voting upgrader`, data);
+    console.log(`2.j.  Multirole: transfer owner role of ${contractName} to voting upgrader`, data);
+  }
+
+  const resetProposerGovernorTx = await governor.populateTransaction.resetMember(1, governorV2.address);
```

Comment:
We cannot do a `governor.populateTransaction.resetMember(1, ZERO_ADDRESS)` so I've decided to assign the role to the `governorV2`

Reply:
If you want to burn the permissioning, you can set it to 0x1 to avoid the 0x0 check. That address is just unlikely for someone to find the private keys.